package.json

```diff
@@ -2,7 +2,7 @@
   "name": "search-insights",
   "description": "Library for reporting click, conversion and view metrics using the Algolia Insights API",
   "version": "2.0.1",
-  "jsdelivr": "dist/search-insights.min.js",
+  "jsdelivr": "dist/search-insights.iife.min.js",
```

Comment:
can we leave it as umd, and let the customer use the specific url for iife?

Reply:
what exactly would be the vaiue of keeping an UMD build for the snippet? I'm struggling to find any.

Reply:
To be honest, I have no idea what kind of side-effects we'll possibly see after replacing umd with iife. So my suggestion is, to keep things as-is for all the customers who have been using umd with no problem and add a little iife for those with problem with umd.

Reply:
agreed, changing from umd to iife could be a bigger change than expected

Reply:
ok, i changed back jsdelivr to serve umd build in 870526a